sdk/identity/Azure.Identity/src/InteractiveBrowserCredentialOptions.cs

```diff
@@ -53,5 +53,10 @@ public string TenantId
         /// Avoids the account prompt and pre-populates the username of the account to login.
         /// </summary>
         public string LoginHint { get; set; }
+
+        /// <summary>
+        /// When set to true the <see cref="InteractiveBrowserCredential"/> will use the system authentication broker instead of the system browser when available.
```

Comment:
```suggestion
        /// When set to `true`, the <see cref="InteractiveBrowserCredential"/> will use the system authentication broker instead of the system browser when available.
```